api/v1/storagecluster_types.go

```diff
@@ -247,8 +247,14 @@ type MonitoringSpec struct {
 // EncryptionSpec defines if encryption should be enabled for the Storage Cluster
 // It is optional and defaults to false.
 type EncryptionSpec struct {
+	// deprecated from OCS 4.10 onwards, acting as a dummy,
+	// UI will keep sending this flag for backward compatibility (OCP 4.10 + OCS 4.9)
 	// +optional
-	Enable               bool                     `json:"enable,omitempty"`
+	Enable bool `json:"enable,omitempty"`
+	// +optional
+	ClusterWide bool `json:"clusterWide,omitempty"`
+	// +optional
+	StorageClassWide     bool                     `json:"storageClassWide,omitempty"`
```

Comment:
```suggestion
	StorageClass     bool                     `json:"storageClassWide,omitempty"`
```

Reply:
This is what we are using from the UI: https://github.com/openshift/console/pull/10563/commits/2385083c7c71001d5b89e2657aa72a264a0f2644#r758951150

Same format I used in backend as well.

Reply:
Ui is not using the name StorageClassWide while displaying the encryption page, so we better stick to that.

Reply:
okay, got it... Will change it to...
	// +optional
	ClusterWide bool `json:"clusterWide,omitempty"`
	// +optional
	StorageClass    bool                     `json:"storageClass,omitempty"`